x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -36,10 +36,61 @@ interface RouterProps {
   store: Store;
 }
 
+interface NavTabs {
+  name: string;
+  display: string;
+  exact: boolean;
+  to: string;
+}
+const navTabs: NavTabs[] = [
+  {
+    name: 'home',
+    display: 'Home',
+    exact: true,
+    to: '/',
+  },
+  {
+    name: 'management',
+    display: 'Management',
+    exact: false,
+    to: '/management',
+  },
+  {
+    name: 'alerts',
+    display: 'Alerts',
+    exact: false,
+    to: '/alerts',
+  },
+  {
+    name: 'policies',
+    display: 'Policies',
+    exact: false,
+    to: '/policy',
+  },
+];
+
+function renderNavTabs(tabs: NavTabs[]) {
+  return tabs.map((tab, index) => {
+    return (
+      <NavLink
```

Comment:
We should probably use the`EuiTabs` component for this and not the `NavLink` from react-router. `EuiTab` allows for `href` to be set as well as a `onClick` callback - the callback would be where you can handle calling react-router's `history.push`.  We might need to `styled` the `EuiTabs` a little (see [example in Ingest](https://github.com/elastic/kibana/blob/12f35d5788f5250803434b6d8f25d9df82ac0940/x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/header.tsx) just committed recently)

Also - one of the thing the EUI team discourages is to use the internal class names on components directly like this :(